### PR TITLE
Rename argument in get_supplemental_claim_upload

### PR DESCRIPTION
**Pull request by kayline**
## Summary

- *This work is behind a feature toggle (flipper): NO*
- was :uuid, now :guid
- to match get_notice_of_disagreement_upload for easier code sharing and shared testing
- Decision Reviews, yes

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/96645

## Testing done

- [x] *New code is covered by unit tests*
- Changed one named argument from :uuid to :guid

## What areas of the site does it impact?
DecisionReviewV1::Appeals::SupplementalClaimServices

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by shaunburdick — APPROVED** [review]
LGTM!

**Review by dfong-adh — APPROVED** [review]

**Review by Mottie — APPROVED** [review]
LGTM

**Review by rjohnson2011 — APPROVED** [review]

**Review by shaunburdick — APPROVED** [review]
Re-approving after merge conflicts resolved

**Review by rjohnson2011 — APPROVED** [review]

**Pull request merged**
